pandas/io/excel.py

```diff
@@ -168,7 +168,20 @@ def get_writer(engine_name):
         raise ValueError("No Excel writer '%s'" % engine_name)
 
 
-@Appender(_read_excel_doc)
+def split_doc(doc_string, word_wrap=80):
```

Comment:
I think the how it is here done is fine. In principle we could add a check for this in Appender, but this is also fine.